lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -77,15 +77,18 @@ def configure_parser(conf)
       end
     end
 
-    def next_token
-      return nil unless File.exist?(@state_file)
-      File.read(@state_file).chomp
+    def state_file_for(log_stream_name = nil)
```

Comment:
Is there any code path `log_stream_name` to be `nil`?

Reply:
ok, it comes from config_param. unlikely to become nil. I'll remove this.